packages/gatsby/src/schema/schema.js

```diff
@@ -174,6 +200,42 @@ const processAddedType = ({ schemaComposer, type, parentSpan }) => {
     }
   }
   schemaComposer.addSchemaMustHaveType(typeComposer)
+
+  typeComposer.setExtension(`createdFrom`, createdFrom)
```

Comment:
Just thinking out loud: is there value in grouping certain extensions under `extensions.metadata`? Hmm probably not because we'd always have to deepmerge

Reply:
Yeah, let's not do that. I don't want to make a huge utility module for that.